src/dotVariant.Generator/templates/Variant.scriban-cs

```diff
@@ -97,20 +97,32 @@ namespace {{ Variant.Namespace }}
         public {{ method_modifiers }}bool Equals({{ variant_nullable_param }} other)
             => {{ if Variant.IsReferenceType; "!(other is null) && "; end }}_variant.Equals(other._variant);
 
+        /// <summary>Returns a value that indicates whether two variants are equal.</summary>
+        /// <param name="lhs">The first <see cref="{{ Variant.Type }}" /> to compare.</param>
+        /// <param name="rhs">The second <see cref="{{ Variant.Type }}" /> to compare.</param>
+        /// <returns><c>true</c> if <paramref name="lhs"/> and <paramref name="rhs"/> are equal; otherwise, false.</returns>
+        /// <seealso cref="Equals({{variant_nullable_param }})" />
```

Comment:
```suggestion
        /// <summary>Compare two {{ Variant.Type }} objects for equality.</summary>
        /// <param name="lhs">The first <see cref="{{ Variant.Type }}" /> to compare.</param>
        /// <param name="rhs">The second <see cref="{{ Variant.Type }}" /> to compare.</param>
        /// <returns><see langword="true" /> if <paramref name="lhs"/> and <paramref name="rhs"/> are considered equal; otherwise, <see langword="false" />.</returns>
        /// <seealso cref="Equals({{variant_nullable_param }})" />
```

Reply:
I construct my summaries with "indicates/indicating whether" because I try to mimick some .Net summaries, but I don't have any preference either way. Except that your version contains the specific type again, which probably is better.
And always forget that langword is a thing.

Reply:
> I construct my summaries with "indicates/indicating whether" because I try to mimick some .Net summaries

I consdier that an anti-pattern. The function does not indicate anything. It perform an action. So I'd rather the summary says what the function _does_.

Reply:
That is why the return value indicates, which I normally use for properties and not methods(probably didn't take the best place to copy from)
https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector2.op_equality?view=net-6.0
Should have taken string, which is much better with
> Determines whether two specified strings have the same value.
here the method actually does something^^